velox/core/PlanNode.cpp

```diff
@@ -569,6 +943,169 @@ WindowNode::WindowNode(
       "Number of sorting keys must be equal to the number of sorting orders");
 }
 
+namespace {
+std::unordered_map<std::string, WindowNode::BoundType> nameToBoundType() {
+  return {
+      {"CURRENT ROW", WindowNode::BoundType::kCurrentRow},
```

Comment:
nit: maybe having enum -> string mapping (and then inverting) would feel more natural - but this is super nit-picking :)